### PR TITLE
vim: Syntax-highlight continue and break statements

### PR DESCRIPTION
**Pull request by nirbheek**

### CODE REVIEW

File: data/syntax-highlighting/vim/syntax/meson.vim [hunk 1]
```diff
@@ -32,8 +32,9 @@ set cpo&vim
 
 " http://mesonbuild.com/Syntax.html
 syn keyword mesonConditional	elif else if endif
-syn keyword mesonRepeat	foreach endforeach
+syn keyword mesonRepeat		foreach endforeach
```
Comment (eli-schwartz) [inline data/syntax-highlighting/vim/syntax/meson.vim:35]: What's the purpose of this diff hunk?

Comment (nirbheek) [reply data/syntax-highlighting/vim/syntax/meson.vim:35]: A tab for editor alignment:
image

Comment (ignatenkobrain) [reply data/syntax-highlighting/vim/syntax/meson.vim:35]: I think depends on tab size :)

Comment (eli-schwartz) [reply data/syntax-highlighting/vim/syntax/meson.vim:35]: Meh. this file contains `" vim:set sw=2 sts=2 ts=8 noet:` so the change is correct and needed if I open it in vim, but git log seems to align tabs including the leading `-+ ` for the diff context. Sorry...

Comment (nirbheek) [reply data/syntax-highlighting/vim/syntax/meson.vim:35]: No worries, thanks for eyeballs :)

**Pull request merged**